tests/nightly/nightly-kickoff.sh

```diff
@@ -17,6 +17,12 @@
 echo "Removing VIC directory if present"
 rm -rf vic
 
+echo "Cleanup logs from previous run"
+rm -rf *.xml 
```

Comment:
`rm -rf` is fine here, but just `rm -f` will suffice since they're not directories.